### ISSUE TITLE
Rename the project to something other than mineflayer-baritone

### ISSUE THREAD

**Issue by mat-1** [mat-1/mineflayer-baritone#9]
Originally my idea was to recreate the Baritone API for Mineflayer but the project is currently very different from that, so I think it should be renamed. Make a comment if you have an idea for a better name because I don't really know what it should be lol

**Comment by DrMoraschi** [comment]
baritone.js? Lol

**Comment by mat-1** [comment]
> 
> 
> baritone.js? Lol

I don't want it to have baritone in the name though

**Comment by DrMoraschi** [comment]
Right, sorry. How about matfinder/mathfinder(mat + pathfinder, get it?) or mineflayer-mathfinder/matfinder xD
